docs/UserGuide.md

```diff
@@ -242,6 +243,19 @@ Format: `add e/<EXERCISE> c/<CALORIES_BURNT_PER_MIN> d/<DURATION_IN_MIN>`
 Sample input: `add e/calisthenics c/10 d/5`     
 Expected output:
 
+---
+### Change user height: `change`
+Allows users to change their height, deletes the old height input and updates their BMI
```

Comment:
Shall we make the wording consistent in the UG?

Something along the lines of `Changes user height and updates the BMI value` is better I feel.